app/src/main/java/org/stepic/droid/util/DbParseHelper.kt

```diff
@@ -3,35 +3,51 @@ package org.stepic.droid.util
 object DbParseHelper {
 
     private const val DEFAULT_SEPARATOR = "__,__"
+    private const val PREFIX = "["
+    private const val POSTFIX = "]"
 
     @JvmStatic
     @JvmOverloads
-    fun parseStringToLongArray(str: String?, separator: String = DEFAULT_SEPARATOR): LongArray? =
-            parseStringToLongList(str, separator)?.toLongArray()
+    fun parseStringToLongArray(str: String?, separator: String = DEFAULT_SEPARATOR, escapeSymbols: Boolean = false): LongArray? =
```

Comment:
let make public escape function that will be used in `lessonDao.removeLike(DbStructureLesson.Columns.COURSES, "%[$courseId]%")` and in parse functions to make code safer